### ISSUE TITLE
iOS - Delete shared data once the migration process is completed.

### ISSUE THREAD

**Issue by dvdchr** [wordpress-mobile/WordPress-iOS#19658]
Refs pbArwn-5BP-p2

Due to the reasons described in the document, we cannot simply wipe everything in the App Group since operational data are being read and written there for App Extensions. 

Here are all the items to be deleted once the migration process is completed:

- Temporary backup file of the database in App Group.
- Copy of the Blogging Reminders schedule file in App Group.
- Copy of WP's user defaults in App Group. Some notes of warning:
  - The copying itself will be a bit tricky because we will be copying WP's user defaults' contents into an existing user defaults in the App Group. importing the user defaults contents from here will require some arrangement, so it doesn't needlessly copy App Extensions' contents into JP's user defaults. The copying task will be handled here: https://github.com/wordpress-mobile/WordPress-iOS/issues/19657.
  - In summary, we can't simply wipe the entire user defaults in the App Group because some contents are used for App Extensions.

**Comment by dvdchr** [comment]
We won't be doing this task. The database backup files will be automatically nuked upon successful restoration.
